core-plugins/src/main/java/co/cask/hydrator/plugin/transform/LogParserTransform.java

```diff
@@ -110,7 +110,13 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws Ille
       }
       Schema.Field inputNameSchema = inputSchema.getField(config.inputName);
       try {
-        if (inputNameSchema != null && inputNameSchema.getSchema() != null) {
+        if (inputNameSchema == null) {
+          throw new IllegalArgumentException(
+            "Invalid inputName, no known inputField matches given input of " + config.inputName);
```

Comment:
Invalid inputName config.inputName. Schema does not contain a field with that name.